server.js

```diff
@@ -2,6 +2,7 @@
 var express =  require('express');
 var app = express();
 app.use(express.static('src'));
+app.use(express.static('dist'));
```

Comment:
We have two static folders? I think we need only one. Here you defined 'src' and also 'dist' as static folder.